EIPS/eip-5753.md

```diff
@@ -0,0 +1,247 @@
+
+However, it should be different for NFTs because NFTs have plenty of use cases that require the NFT to stay in the holder's wallet even when it is used as collateral for a loan. You may want to keep using your NFT as a verified PFP on Twitter, or use it to authorize a Discord server through collab.land. You may want to use your NFT in a P2E game. And you should be able to do all of this even during the lending period, just like you are able to live in your house even if it is mortgaged.
+
+There are plenty of other use cases, such as:
+
+* Lending/borrowing NFT without a need for collateral
+* Paying for NFT by installments
+* Safe and convenient usage with hot wallets
+* Non-custodial staking and much more. 
+
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+EIP-721 compliant contracts MAY implement this EIP to provide standard methods of locking and unlocking the token at its current owner address. 
+If the token is locked, the `getLocked` function MUST return an address that is able to unlock the token.
+For tokens that are not locked, the `getLocked` function MUST return `address(0)`.
+The user MAY permanently lock the token by calling `lock(address(1), tokenId)`.
+
+When the token is locked, all the [EIP-721](./eip-721.md) transfer functions MUST revert, except transaction has been initiated by an unlocker.
+When the token is locked, the [EIP-721](./eip-721.md) `approve` method MUST revert.
+When the token is locked, the [EIP-721](./eip-721.md) `getApproved` method MAY return address(0).
+When the locked token is transferred by an unlocker, the token MUST be unlocked after the transfer.
+
+Marketplaces should call `getLocked` method of an EIP-721 Lockable token contract to learn whether a token with a specified tokenId is locked or not. Locked tokens SHOULD NOT be available for listings. Locked tokens can not be sold. Thus, marketplaces SHOULD hide the listing for the tokens that has been locked, because such orders can not be fulfilled.  
+
+### Contract Interface
+```solidity
+pragma solidity >=0.8.0;
+
+/// @dev Interface for the Lockable extension
+
+interface ILockable {
+
+    /**
+     * @dev Emitted when `id` token is locked, and `unlocker` is stated as unlocking wallet.
+     */
+    event Lock (address indexed unlocker, uint256 indexed id);
+
+    /**
+     * @dev Emitted when `id` token is unlocked.
+     */
+    event Unlock (uint256 indexed id);
+
+    /**
+     * @dev Locks the `id` token and states `unlocker` wallet as unlocker.
+     */
+    function lock(address unlocker, uint256 id) external;
```

Comment:
Somewhere in the Specification section, you should mention that a locked token can't be relocked.

Should also clarify the behaviour if `lock` is called with the same `unlocker` while locked.